index.html

```diff
@@ -0,0 +1,39 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="style.css">
+    <title>Awesome Book</title>
+  </head>
+  <body>
+    <header>
+      <h1>Awesome Books</h1>
+      <br />
+    </header>
+    <section class="books">
+      <div class="books_list" id="books_list"></div>
+      <br />
+
+      <form class="bookInfo" id="bookInfo">
+        <input
+          class="title"
+          name="bookTitle"
+          type="text"
+          placeholder="Title"
+          required /><br /><br />
+        <input
+          class="author"
+          name="bookAuthor"
+          type="text"
+          placeholder="Author"
+          required /><br /><br />
+        <button type="submit">Add</button>
+      </form>
+    </section>
+
+    <!-- <script src="./scriptv2.js"></script> -->
```

Comment:
- [OPTIONAL] Kindly remove the commented code as to make your code clean.

Reply:
okey